src/main/scala/cromwell/singleuser/SingleWorkflowRunner.scala

```diff
@@ -0,0 +1,26 @@
+package cromwell.singleuser
+
+import java.io.File
+
+import cromwell.Main
+import cromwell.binding.WdlBinding
+import cromwell.server.WorkflowManagerSystem
+
+import scala.util.{Failure, Success, Try}
+
+// FIXME: Once we have a definition of what 'run' means in Main.scala beyond some pretty printing, it goes here
+object SingleWorkflowRunner extends WorkflowManagerSystem {
+
+  // FIXME: This function is only here temporarily, it'll never really exist in this form (or this name)
+  def runIt(args: Array[String]): Unit = {
+    if (args.isEmpty) Main.usageAndExit()
+    else {
+      Try(WdlBinding.process(new File(args(1)))) match {
+        case Success(b) => Main.printProcessedBinding(b)
+        case Failure(e) =>
+          println(e)
```

Comment:
I thought I had a comment somewhere, I suspect in my accidental push yesterday about these. the printlns are all over the original main() code and I didn't feel like figuring out the right thing to do yet. I'm going to continue to punt but yes, I agree